easybuild/easyconfigs/s/SLEPc/SLEPc-3.20.1-foss-2023b.eb

```
@@ -0,0 +1,21 @@
+name = 'SLEPc'
+version = '3.20.1'
+
+homepage = 'https://slepc.upv.es'
+description = """SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution
+ of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for
+ either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a
+ partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['https://slepc.upv.es/download/distrib']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5a36b664895881d3858d0644f56bf7bb922bdab70d732fa11cbf6442fec11806']
+
+dependencies = [('PETSc', '3.20.3')]
```

This should also depend on the merged 3.22.5 easyconfig:
```suggestion
dependencies = [('PETSc', '3.22.5')]
```

And I don't know how PETSc / SLEPc work together (they have slightly similar version numbers), but maybe SLEPc itself should then also be bumped to 3.22?

---

Latest 3.22 version of SLEPc is 3.22.2

---

Ok, let's bump SLEPc also (to 3.22.2)